.rdoc_options

```diff
@@ -0,0 +1,8 @@
+exclude:
+  - lib/rdoc/rd/block_parser.ry
+  - lib/rdoc/rd/inline_parser.ry
+  - lib/rdoc/markdown.kpeg
+  - lib/rdoc/markdown/literals.kpeg
+op_dir: _site # for GH pages and should match the config of RDoc task in Rakefile
```

Comment:
Minor: by using the casing in the branding, the reader has a higher chance to differentiate what kinds of pages we are referring to

```suggestion
op_dir: _site # for GitHub Pages and should match the config of RDoc task in Rakefile
```